addons/web_editor/static/src/js/backend/fields.js

```diff
@@ -83,12 +83,45 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend(TranslatableFieldMi
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * Return the domain for attachments used in media dialog.
+     * We look for attachments related to the current document. If there is a value for the model
+     * field, it is used to search attachments, and the attachments from the current document are
+     * filtered to display only user-created documents.
+     * In the case of a wizard such as mail, we have the documents uploaded and those of the model
+     *
+     * @private
+     * @returns {Array} "ir.attachment" odoo domain.
+     */
+    _getAttachmentsDomain: function () {
+        var domain = ['|', ['id', 'in', _.pluck(this.attachments, 'id')]];
+        if (this.recordData.model) {
```

Comment:
Il they are model in the record (for eg: mail, we use this condition).  Do you prefer that I list the modules that have this behavior?

Reply:
I don't know, is there any other module than mail where we need this ? Are you in GR ? (Could you come or can I come so that we speak about this again ?)